### ISSUE TITLE
get avatar for non-list contact - getProfilePicFromServer throw error

### ISSUE THREAD

**Issue by antirek** [wppconnect-team/wppconnect#493]
## Description

I want check number in whatsapp - exist or no? If exist I want receive avatar. 
I'm sure that avatar exist because I see it on other whatsapp account.

## Environment

- **Wppconnect version(s):** [1.8.13]
- **Browser:** [Chromium 88]
- **OS:** [Ubuntu 20.04]
- **Node version:** [Node 14]
- **WhatsApp version:** [2.2132.6, 2.2134.9]
- **MultiDevice (BETA):**no

## Log Output
```
ModelCreateError: Unknown
    at new i (https://web.whatsapp.com/bootstrap_qr.abbc2ea8366e2cd9a145.js:2:403072)
    at new k (https://web.whatsapp.com/bootstrap_qr.abbc2ea8366e2cd9a145.js:56:209667)
    at https://web.whatsapp.com/bootstrap_qr.abbc2ea8366e2cd9a145.js:55:335232
    at V (https://web.whatsapp.com/bootstrap_qr.abbc2ea8366e2cd9a145.js:2:373738)
    at MutationObserver.K (https://web.whatsapp.com/bootstrap_qr.abbc2ea8366e2cd9a145.js:2:373319)

```

## Your Code

```
const profilePicThumbObj = await client.getProfilePicFromServer(chatId as string);

```

**Comment by antirek** [comment]
@edgardmessias please check and merge #497

**Comment by edgardmessias** [comment]
Closed by #497
